models/twilio__account_overview.sql

```diff
@@ -15,9 +21,13 @@ usage_record as (
 
 select
     messages.account_id,
+    account_history.friendly_name as account_name,
+    account_history.status as account_status,
+    account_history.type as account_type,
     messages.date_day,
     messages.date_week,
     messages.date_month,
+    messages.price_unit,
```

Comment:
For maintenance and ease of understanding, I wanted to keep the group by fields all together.